framework/core/js/src/forum/components/UserPage.tsx

```diff
@@ -159,6 +159,20 @@ export default class UserPage<CustomAttrs extends IUserPageAttrs = IUserPageAttr
       );
     }
 
+    if (app.session.user!.id() === user.id() || app.forum.attribute<boolean>('canModerateAccessTokens')) {
+      if (app.session.user!.id() !== user.id()) {
+        items.add('separator', <Separator />, -90);
```

Comment:
We should try and be more specific to prevent clashing of keys with extensions

```suggestion
        items.add('security-separator', <Separator />, -90);
```